### PR TITLE
New example with multiple execution blocks

### PR DESCRIPTION
**Pull request by wimdeblauwe**
This pull request adds a new example with multiple execution blocks to generate the following:
* Technical documentation in HTML format
* User manual in PDF format

Sorry about the multiple commits, I tried to squash them into 1 but I guess I did something wrong.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -18,6 +18,7 @@
     </properties>
 
     <modules>
+        <module>asciidoc-multiple-inputs-example</module>
```
Comment (mojavelinux) [inline pom.xml]: Could you move the asciidoc-multiple-inputs-example module to just above the docbook-pipeline-docbkx-example?

**Comment by mojavelinux** [comment]
Thanks for adding this example!

You should be able to squash using `git rebase -i origin/master`. Once that's completed, you'll need to force push to your branch since it rewrites the history of the pull request.

Here's a good reference for git rebase: https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase-i

**Comment by wimdeblauwe** [comment]
Please forget about this pull request. I have created a new one: https://github.com/asciidoctor/asciidoctor-maven-examples/pull/8

The rebasing thing is not working. Since I already pushed my changes to my remote, I cannot rebase (Or I am missing something, but I always get even more commits instead of less)

**Comment by mojavelinux** [comment]
Sometimes, the best way to go about it is just to start from the beginning ;)

**Comment by abelsromero** [comment]
I do a lot of 'force' commits to fix stuff :P 
Git can be a pain sometimes.

**Comment by mojavelinux** [comment]
I won't argue that, git certainly gets the gears turning in the head. The advice I always give to help make sense of it is to think of git as just a tool for moving around patches. Once you see it that way, no command seems far from its function.

**Pull request closed without merging**